tests/ipv6-evasion/ipv6-dos-with-ext-headers-1/README.md

```diff
@@ -0,0 +1,11 @@
+# Description
+
+Test detection of DoS attack that tries to increase decoding effort by including a huge amount of unknown options for Hop-by-Hop Options Header.
+
+# PCAP
+
+Pcap from https://redmine.cs.uni-potsdam.de/projects/pcap/files
+
+# Notes
+
+It might be better to have a dedicated rule that focuses on the DoS aspect in combination with the decoder-event
```

Comment:
I think that the DOS aspect is rather generating a huge amount of alerts...

Reply:
It might be "solved" by a rule with a proper threshold.